translation-bounties-2021.md

```diff
@@ -0,0 +1,33 @@
+# Translation Bounties Program (2020) — Archived
+
+**We want Sora to speak the languages of the world and unite it.**
+
+For that, we started our translation program – you can translate, review translations and receive VAL tokens as a reward for your help.
+We use [Lokalise](https://lokalise.com/) for translations and it makes it really easy to translate.
+Translation includes Fearless Wallet, Sora project translation and Capital project translation (it is a module for Sora iOS application).
+
+[Translation Round Q1 2021 Article](https://sora-xor.medium.com/rewards-for-sora-ecosystem-translations-c69bc38e931a)
+
+### Here is how it works:
+1. Join our [Translation Group on Telegram](https://t.me/sora_translation_community)
+2. Check out the table below – if you want to translate or review a language that is not yet translated/ reviewed – let us know about it in the group.
+We will check and ask you to send us your email and name to add you to Lokalise. Don't hesitate to let us know if you can't access some features you need.
+3. You'll receive invitation and can start translating/ reviewing.
+The language will be assigned to you but **please note it can be reassigned if the translation has not been in progress for some time.**
+4. After translation is complete, we will need to review it before reward is sent out.
+The translator receives 90% of the reward, 10% goes to the reviewer.
+Please note that review is the process of checking (and small corrections) and not major editing – revisions are counted as some part of the translation.
+Who makes revisions – original translator or the reviewer – should be discussed between them or in the group.
+*In some cases it might take some time to find a reviewer, so please be patient – reward will be sent when review is completed.*
+5. After the translation is complete (for Sora only together with Capital – it is a Sora component for iOS app) and reviewed, the award will be assigned – we'll ask for your wallet address in Telegram and add it to the table here.
```

Comment:
Should we mention all projects? After the translation is complete for Fearless or Sora (including Capital), or both

Reply:
what I meant here that in case of Sora it will be complete only together with Capital. Because in general, the projects can be translated separately

Reply:
tried to clarify this in the current version